SConstruct

```diff
@@ -553,6 +553,20 @@ if selected_platform in platform_list:
     if env.dev_build:
         print("NOTE: Developer build, with debug optimization level and debug symbols (unless overridden).")
 
+    # Set x86 CPU instruction sets to use by the compiler's autovectorization.
+    # Only done on GCC/Clang/MinGW, as MSVC does not have enough granularity to enforce SSE4.2
+    # (it can only enforce SSE2, AVX or AVX2).
```

Comment:
It may be possible using undocumented flags, but that may be risky to rely on: https://stackoverflow.com/questions/64053597/how-do-i-enable-sse4-1-and-sse3-but-not-avx-in-msvc

Reply:
Here are Windows x86_64 release template binaries compiled with MSVC 2022:

- Binary with standard options (same as `master`): [godot.windows.template_release.x86_64.master.exe.zip](https://github.com/godotengine/godot/files/12778589/godot.windows.template_release.x86_64.master.exe.zip)
- Binary with `/d2archSSE42` (10 KB smaller): [godot.windows.template_release.x86_64.exe.zip](https://github.com/godotengine/godot/files/12778591/godot.windows.template_release.x86_64.exe.zip)

Reply:
> It may be possible to enforce SSE4.2 using undocumented flags, but that may be risky to rely on: https://stackoverflow.com/a/69328426

Now it complains in CI. Maybe as it is undocumented flag it means that it really didn't pass test of time and should be avoided